tests/testthat/test-rstanarm.R

```diff
@@ -1,8 +1,8 @@
 .runThisTest <- Sys.getenv("RunAllinsightTests") == "yes"
-
+.runThisTest <- TRUE
```

Comment:
why did you set this to TRUE?

Reply:
Oops!

I had set it temporarily for the tests to be run on my laptop but forgot to remove it.

Otherwise these tests are just skipped on my Mac.

Reply:
I have reverted this change in `master` already.